index.html

```diff
@@ -2449,6 +2695,254 @@ <h3 id=new-session><dfn data-lt="new sessions|creating a new session">New Sessio
 </ol>
 </section> <!-- /New Session -->
 
+<section>
+<h3><dfn>Get Session</dfn></h3>
+
+<table class="simple jsoncommand">
+ <tr>
+  <th>HTTP Method</th>
+  <th>URI Template</th>
+ </tr>
+ <tr>
+  <td>GET</td>
+  <td>/session/{session id}</td>
+ </tr>
+</table>
+
+<p>The <a>remote end steps</a> are:
+
+<ol>
+ <li><p>Let <var>body</var> be a JSON <a>Object</a> initialised with:
+  <dl>
+    <dl>
+      <dt>"<code>sessionId</code>"
+      <dd>The <a>session id</a> of the <a>current session</a>.
+
+      <dt>"<code>capabilities</code>"
+      <dd>The capabilities of the <a>current session</a>.
+
+      <dt>"<code>capabilitiesRequest</code>"
```

Comment:
Drop `capabilitiesRequest`

The shape of this object should match the shape of the object returned by `New Session`. The local end is aware of the capabilities that were requested, and (more importantly) this might be absolutely gigantic if someone has include one or more browser profiles.